File: problem-1/c-cpp/problem-1.c
#include <stdio.h>

int main() {
    // Sum of multiples of 3, using Gauss Summation
	int multiples_of_3_sum = 3 * ((333 * (1 + 333)) / 2);

	// Sum of multiples of 5, using Gauss Summation
	int multiples_of_5_sum = 5 * ((199 * (1 + 199)) / 2);

	// Sum of multiples of 15, using Gauss Summation
	int multiples_of_15_sum = 15 * ((66 * (1 + 66)) / 2);

	// Add summations of 3 and 5 then subtract the numbers that have both as factors (multiples of 15)
	int multiples_of_3_and_5 = multiples_of_3_sum + multiples_of_5_sum - multiples_of_15_sum;
    
    // Display result
    printf("%d\n", multiples_of_3_and_5);
    return 0;
}

File: src/com/github/problem1/problem1.java
package com.github.problem1;

public class problem1 {
    public static void main(String[] args) {
        // Sum of multiples of 3, using Gauss Summation
        int multiples_of_3_sum = (333 * (3 + 999)) / 2;

    	// Sum of multiples of 5, using Gauss Summation
        int multiples_of_5_sum = (199 * (5 + 995)) / 2;

        // Sum of multiples of 15, using Gauss Summation
        int multiples_of_15_sum = (66 * (15 + 990)) / 2;
    
    	// Add summations of 3 and 5 then subtract the numbers that have both as factors (multiples of 15)
	    int multiples_of_3_and_5 = multiples_of_3_sum + multiples_of_5_sum - multiples_of_15_sum;

        // Display result
        System.out.printf("%d\n", multiples_of_3_and_5);
    }
}